getting-started/building-your-implementation-plan.md

```diff
@@ -0,0 +1,99 @@
+
+
+## Resources to evaluate and choose the Implementation Methods
+
+Nosto provides [plugins for the most common eCommerce platforms](https://docs.nosto.com/techdocs/implementing-nosto/implement-on-your-website#choosing-the-implementation-method) like Shopify, Magento 2, Shopware 6, BigCommerce and Prestashop.
+
+We recommend using our plugins as a base since they follow platform- and Nosto-specific best practices, save you lots of development time and have been tested in hundreds of eCommerce sites.
+
+### Nosto Module Types and Frontend Implementation Methods
+
+Every Nosto module falls into one of two module types which have their own set of possible implementation methods:
+
+1. **Campaign Widgets:** Product Recommendations, Dynamic Bundles and Onsite Content Personalization (via placements, see previous section, points 3 and 4)
+2. **Listings:** Search and Category Merchandising (replacing native content (Autocomplete/Search Preview, SERPs, PLPs), see previous section, point 5)
+
+### Project Plan Checklist
+
+All of the following questions can be answered with our general and platform-specific documentation. For custom builds and advanced implementations you find additional playbooks below. Our onboarding team is happy to assist you with the evaluation and planning.
+
+- [ ] **Product Data Sync:** How do you send Nosto the product catalog?
+- [ ] **Multi-Currency and Localization:** How do you send Nosto the localized data? How do you make sure the correct language and currency shows in the frontend?
+- [ ] **Customer Group Visibility/Pricing:** How do you handle different product availability or prices depending on customer groups? If you are using SKUs/variants/child products or also have multiple currencies, you can't use Nosto's built-in functionality. We then recommend either an API implementation or fetching the prices via an frontend API.
+- [ ] **Order Data Sync:** How do you send Nosto the placed orders?
+- [ ] **Frontend Script:** How do you inject the Nosto script?
+- [ ] **Session Tagging:** How do you tag the currently logged in user and shopping cart content on all pages?
+- [ ] **Frontend Page Tagging:** How do you tag the different page types?
+- [ ] **Frontend Module Injection:** Which implementation method fits your tech stack and client needs best? (Comparison table below)
+  - [ ] Campaign Widgets: `Nosto-hosted` | `Client API` | `Server API`
+  - [ ] Listings: `Nosto-hosted` | `Client API` | `Server API`
+- [ ] **Custom Event Tracking:** How do you signal Nosto non-standard user interactions like "viewed product" or "added product to cart" events? You need this if you have features like:
+  - [ ] "Add to cart" button on product cards, 
+  - [ ] "Quick View"/"Quick Buy" modals in campaign widgets or listings,
+  - [ ] "Mini-Cart" campaign widgets, 
+  - [ ] ...
+- [ ] **Fallback Plan:** How does your frontend behave in case the Nosto service is unavailable? (The insurance we want you to never need.)
```

Comment:
Less spooky:

```suggestion
- [ ] **Fallback Plan:** How does your frontend behave in case the Nosto service is temporarily unavailable? (The insurance we want you to never need.)
```